examples/simple_ds402_node.py

```diff
@@ -0,0 +1,160 @@
+import canopen
+import sys
+import os
+import traceback
+
+from canopen import Node402, Network
+
+import time
+
+try:
+
+    # Start with creating a network representing one CAN bus
+    network = canopen.Network()
+
+    # Connect to the CAN bus
+    network.connect(bustype='kvaser', channel=0, bitrate=1000000)
+
+    network.check()
+
+    # Add some nodes with corresponding Object Dictionaries
+    node = Node402(35, '/home/andre/Code/test/jupiter.eds')
+    network.add_node(node)
+    # network.add_node(34, '/home/andre/Code/test/jupiter.eds')
+    # node = network[34]
+
+    # Reset network
+    # node.nmt.state = 'RESET COMMUNICATION'
+    node.nmt.state = 'RESET'
+    node.nmt.wait_for_bootup(15)
+
+    print 'node state 1) = {0}'.format(node.nmt.state)
+
+    # Iterate over arrays or records
+    error_log = node.sdo[0x1003]
+    for error in error_log.values():
+        print("Error {0} was found in the log".format(error.raw))
+
+    for node_id in network:
+        print(network[node_id])
```

Comment:
You're getting this since you are mixing print functions and print statements. You should use print functions only as print statements only work on Python 2.7.